mofa/simulation/graspa.py

```diff
@@ -0,0 +1,188 @@
+from dataclasses import dataclass
+import ase
+from pathlib import Path
+import numpy as np
+import subprocess
+import os
+import platform
+
+_file_dir = Path(__file__).parent / 'files' / 'graspa_template'
+
+class gRASPARunner:
```

Comment:
Mark this class as a "dataclass" so that we can set the `graspa_comamnd` and `run_dir` fields on creation, like we do for CP2K: https://github.com/globus-labs/mof-generation-at-scale/blob/main/mofa/simulation/cp2k.py#L108